src/api/route-services/__mocks__/projects.js

```diff
@@ -0,0 +1,24 @@
+const mockUpdateProject = jest.fn((projectUuid, body) => {
+  if (!projectUuid
+    || !body
+  ) {
+    throw new Error('Invalid body');
+  }
+
+  return Promise.resolve({
+    name: 'Test project',
+    description: '',
+    createdDate: '',
+    lastModified: '',
+    uuid: 'project-1',
```

Comment:
Shouldn't this be the `projectUuid` that we got as a parameter?

Reply:
No, the `projectUuid` inside a `project` is only `uuid`, so that you wouldn't be writing `project.projectUuid`.